spec/yaml/swiftnav/sbp/gnss.yaml

```diff
@@ -45,6 +45,9 @@ definitions:
                       - 13: BDS2 B2
                       - 14: GAL E1B
                       - 20: GAL E7I
+                      - 31: QZS L1CA
```

Comment:
Referring to all the constellations here (not QZSS): Is there a reason why only these signal codes were added and not all of them?